src/components/ContractDeploy/ContractDeploy.tsx

```diff
@@ -7,95 +7,63 @@ import { setCallResult } from "../../actions";
 import { Button } from "../common/ui";
 
 interface IProps {
-  contractName: string;
   bytecode: any;
   abi: any;
   vscode: any;
-  compiled: any;
-  error: Error | null;
   gasEstimate: number;
   deployedResult: string;
-  compiledResult: object;
-  callResult: object;
-  deployAccount: IAccount;
+  compiledResult: any;
```

Comment:
Compiled result is output from the compiler. The spec can be found at:

* https://solidity.readthedocs.io/en/v0.7.2/using-the-compiler.html#output-description